exercises/concept/castle-dinner/.docs/instructions.md

```diff
@@ -2,18 +2,18 @@
 
 Your majesty is hosting a dinner party in the [Kalmar Unions][kalmar_union] capital Copenhagen.
 It is the biggest party of the year and all the nobles are invited.
-But rumors are spreading across the country that there is a plot to poison your majesty.
+But rumors are spreading across the country that there is a plot to poison Your Majesty.
 
-Her majesty has asked you, the royal poison finder, to find out if any of the food or drinks are poisoned.
+Her majesty has asked you, the royal poison finder, to find out if any food or drinks are poisoned.
 Your majesty is planning to serve only **Mushroom pasties** as food, but for drinks, there are a variety of drinks.
 
 ## 1. Check if the food is correct
 
-There is a high chance that the food is poisoned if it is not the food that her majesty ordered.
-If the food is not the food that her majesty ordered, then it should be tossed out.
+There is a high chance that the food is poisoned if it is not what her majesty ordered.
+If the food is not what her majesty ordered, it should be tossed out.
 
 Implement the `check_food?` method, which takes the argument `food` which holds the name of the food as a `String`.
-The method should return the food if the food is `"Mushroom pasties"`, otherwise it should return `nil`.
+The method should return the food if it is' Mushroom pasties`; otherwise, it should return `nil`.
```

Comment:
```suggestion
The method should return the food if it is `Mushroom pasties`; otherwise, it should return `nil`.
```

I'm not sure this is what was intended, but something is not right.